config/locales/sv.yml

```diff
@@ -485,7 +485,7 @@ sv:
       news: nyheter
       government: regering
       christian: kristen
-      humor: humör
+      humor: humor
```

Comment:
You shouldn't edit sv.yml. Humör is the correct spelling in Swedish.

Reply:
Sorry this was me just fixing a spelling mistake in the Swedish file. I thought this might be committed soon

Reply:
@alexiamcdonald I see I accidentally reverted your change when I attempted to resolve the merge conflicts. I'll set it back to "humor"